tests/lib.rs

```diff
@@ -210,7 +210,7 @@ fn control_fdr_ev(test: &str) {
     libprosic::estimation::fdr::ev::control_fdr(
         &mut calls,
         &mut writer,
-        &libprosic::SimpleEvent { name: "SOMATIC".to_owned() },
+        &vec![ libprosic::SimpleEvent { name: "SOMATIC".to_owned() } ],
```

Comment:
This can be a constant array, i.e.: `&[ libprosic::SimpleEvent { name: "SOMATIC".to_owned() } ]`.